src/main/java/at/hannibal2/skyhanni/features/mining/ColdOverlay.kt

```diff
@@ -0,0 +1,64 @@
+package at.hannibal2.skyhanni.features.mining
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.HypixelData
+import at.hannibal2.skyhanni.data.ScoreboardData
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.features.gui.customscoreboard.ScoreboardPattern
+import at.hannibal2.skyhanni.utils.StringUtils.matchFirst
+import at.hannibal2.skyhanni.utils.StringUtils.matches
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import io.github.moulberry.notenoughupdates.util.Utils
+import net.minecraft.client.Minecraft
+import net.minecraft.client.gui.ScaledResolution
+import net.minecraft.client.renderer.GlStateManager
+import net.minecraft.util.ResourceLocation
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import org.lwjgl.opengl.GL11
+
+
+class ColdOverlay {
+
+    private val areaPattern by RepoPattern.pattern("mining.coldoverlay.area", "Glacite Tunnels")
+
+    fun getCold(): Int? = ScoreboardData.sidebarLinesFormatted.matchFirst(ScoreboardPattern.coldPattern) {
+        return group("cold").replace("-", "").toInt()
```

Comment:
just a nitpick, i would do 
```suggestion
        return group("cold").toInt().absoluteValue
```

Reply:
My idea: move a cold value into clacite tunnel api, and then use ColdChangeEvent instead. (we use it in scoreboard, custom mining notifications, and now here as well)

Reply:
> glacite tunnel api
what if we just did a general mining api instead?

Reply:
> > glacite tunnel api
> 
> what if we just did a general mining api instead?

sounds better, I agree

Reply:
There is already a FossilExcavatorAPI, not sure if we want to move everything into one big mining api. but since we dont have one yet, we might start with one already. we can always split it up later if it gets too big.